tests/test_class_import.py

```diff
@@ -130,6 +131,14 @@ def test_fake_strftime_function():
     assert fake_strftime_function() == '2012'
 
 
+@freeze_time("2022-01-01")
+def test_fake_time_function_as_class_attribute():
```

Comment:
It is great that you added tests, but it is also important to explain on `README` the new optional usage in English so more people can easily reuse your solution =)

Reply:
You can also add some description of this change on `CHANGELOG`.